tests/mobly/base_test_test.py

```diff
@@ -20,6 +20,10 @@
 from mobly import config_parser
 from mobly import signals
 
+from tests.mobly import records_test
```

Comment:
I don't believe you'll be able to import this as tests depending on other tests is not allowed. That's why we added the 'lib' module.